boa3_test/test_drive/model/network/payloads/testblock.py

```diff
@@ -35,7 +35,7 @@ def to_json(self) -> Dict[str, Any]:
         return json_block
 
     @classmethod
-    def from_json(cls, json: Dict[str, Any]) -> TestBlock:
+    def from_json(cls, json: Dict[str, Any], *args, **kwargs) -> TestBlock:
```

Comment:
Why add `*args, **kwargs` if they are not being used? Is it to just standardize the `from_json` method?